src/api/index.js

```diff
@@ -300,6 +300,40 @@ export async function fetchMarkRepoNotificationAsRead(
   return response;
 }
 
+export async function fetchNotificationsCount(accessToken) {
+  const ENDPOINT = `${root}/notifications?per_page=1`;
+  const response = await fetch(ENDPOINT, accessTokenParameters(accessToken));
+
+  let linkHeader = response.headers.get('Link');
+  let count = 0;
+
+  if (linkHeader) {
+    linkHeader = linkHeader.match(/page=(\d)+/g).pop();
+    count = linkHeader.split('=').pop();
+  }
+
+  return count;
+}
+
+export async function fetchRepoNotificationsCount(
+  owner,
+  repoName,
+  accessToken
+) {
+  const ENDPOINT = `${root}/repos/${owner}/${repoName}/notifications?per_page=1`;
+  const response = await fetch(ENDPOINT, accessTokenParameters(accessToken));
+
+  let linkHeader = response.headers.get('Link');
+  let count = 1;
+
+  if (linkHeader) {
+    linkHeader = linkHeader.match(/page=(\d)+/g).pop();
+    count = linkHeader.split('=').pop();
+  }
+
+  return +count;
```

Comment:
what do `+count` do ? Oo

Reply:
This convert the value to a number, similarly does the call to `parseInt()`

Reply:
Nice, thanks